examples/green_screen/main.cpp

```diff
@@ -0,0 +1,529 @@
+                                 const k4a_device_configuration_t &main_config,
+                                 const k4a_device_configuration_t &secondary_config,
+                                 const cv::Size &chessboard_pattern,
+                                 float chessboard_square_length)
+{
+    k4a::calibration main_calibration = capturer.get_master_device().get_calibration(main_config.depth_mode,
+                                                                                     main_config.color_resolution);
+    k4a::calibration secondary_calibration =
+        capturer.get_subordinate_device_by_index(0).get_calibration(secondary_config.depth_mode,
+                                                                    secondary_config.color_resolution);
+    vector<vector<cv::Point2f>> main_chessboard_corners_list;
+    vector<vector<cv::Point2f>> secondary_chessboard_corners_list;
+    while (true)
+    {
+        vector<k4a::capture> captures = capturer.get_synchronized_captures({ secondary_config }); // requires vector
+        k4a::capture &main_capture = captures[0];
+        k4a::capture &secondary_capture = captures[1];
+        // get_color_image is guaranteed to be non-null because we use get_synchronized_captures for color
+        // (get_synchronized_captures also offers a flag to use depth for the secondary camera instead of color).
+        k4a::image main_color_image = main_capture.get_color_image();
+        k4a::image secondary_color_image = secondary_capture.get_color_image();
+        cv::Mat cv_main_color_image = color_to_opencv(main_color_image);
+        cv::Mat cv_secondary_color_image = color_to_opencv(secondary_color_image);
+
+        vector<cv::Point2f> main_chessboard_corners;
+        vector<cv::Point2f> secondary_chessboard_corners;
+        bool got_corners = find_chessboard_corners_helper(cv_main_color_image,
+                                                          cv_secondary_color_image,
+                                                          chessboard_pattern,
+                                                          main_chessboard_corners,
+                                                          secondary_chessboard_corners);
+        if (got_corners)
+        {
+            main_chessboard_corners_list.emplace_back(main_chessboard_corners);
+            secondary_chessboard_corners_list.emplace_back(secondary_chessboard_corners);
+            cv::drawChessboardCorners(cv_main_color_image, chessboard_pattern, main_chessboard_corners, true);
+            cv::drawChessboardCorners(cv_secondary_color_image, chessboard_pattern, secondary_chessboard_corners, true);
+        }
+
+        cv::imshow("Chessboard view from main camera", cv_main_color_image);
+        cv::waitKey(1);
+        cv::imshow("Chessboard view from secondary camera", cv_secondary_color_image);
+        cv::waitKey(1);
+
+        // Get 20 frames before doing calibration.
```

Comment:
Is this comment right? code looks like its validating the number of corners, not the number of frames.